plane/source/docs/precision-autolanding.rst

```diff
@@ -0,0 +1,136 @@
+If not, you're coming in too steep for the negative-thrust-to-mass ratio of your aircraft.
+
+.. tip::
+
+   Keep in mind that whatever value you determine as your maximum may
+   not be acceptable in all wind conditions. It is best to be a little
+   conservative to maintain repeatability.
+
+Setting up a Pre-Flare
+======================
+
+With a rangefinder or accurate GPS, and airspeed sensors installed, a pre-flare point can be set since we will have an accurate airspeed and altitude reading. 
+
+This gives us a good idea of our momentum and stable "initial conditions" to the final flare. Set :ref:`LAND_PF_ALT<LAND_PF_ALT>` (and/or :ref:`LAND_PF_SEC<LAND_PF_SEC>`) to a fairly high point (for example 10m) and adjust from there. Next set :ref:`LAND_PF_ARSPD<LAND_PF_ARSPD>` to a value just above your stall speed.
+
+When :ref:`LAND_PF_ALT<LAND_PF_ALT>` is reached the airspeed demand will instantly go from :ref:`TECS_LAND_ARSPD <TECS_LAND_ARSPD>` to :ref:`LAND_PF_ARSPD<LAND_PF_ARSPD>`.
+
+This will cause it to slam on the brakes if reverse thrust is being used(see :ref:`reverse-thrust-autolanding`)so that the airspeed reduces to the desired airspeed quickly, otherwise plan for some time/distance for the airspeed to reduce.
+
+The trick is to set :ref:`LAND_PF_ALT<LAND_PF_ALT>` to an altitude where it
+achieves :ref:`LAND_PF_ARSPD<LAND_PF_ARSPD>` before killing the throttle at
+:ref:`LAND_FLARE_ALT<LAND_FLARE_ALT>` (which is set at a lower altitude - around 1
+or 2m).
+
+Example, :ref:`TECS_LAND_ARSPD <TECS_LAND_ARSPD>` = 15, :ref:`LAND_PF_ARSPD<LAND_PF_ARSPD>` = 12, :ref:`LAND_PF_ALT<LAND_PF_ALT>` = 12, :ref:`LAND_FLARE_ALT<LAND_FLARE_ALT>` = 2. Depending on your slope, mass of aircraft and motor+propeller thrust ability, you're expecting the aircraft to decelerate from 15m/s to 12m/s airspeed while dropping 10m to 2m. These are the critical params to adjust to ensure a smooth and slow flare below 2m altitude.
+
+Flare
+-----
+
+Now that you are starting the flare with a stable and predictable airspeed, it's much easier to :ref:`control the flare <automatic-landing_controlling_the_flare>`. 
+If you've already tuned your flare for an auto-land without reverse thrust you'll want to retune it. 
+You'll notice you're coming in much slower ad tuning will be easier. 
+The tweaks and compromises you had to do before are much easier to deal with.
+
+
+Determining actual stall speed of your aircraft
++++++++++++++++++++++++++++++++++++++++++++++++
+
+Unless you really know what you're doing, stall speed can be hard to estimate. 
+Traditionally, to determine this true value you would need to slowly decrease your airspeed until you stall but that comes with the pesky problem that now you have a stalled aircraft falling out of the sky.
+
+With :ref:`LAND_PF_ALT<LAND_PF_ALT>` and :ref:`LAND_PF_ARSPD<LAND_PF_ARSPD>` you can check your stall speed much lower to the ground. 
```

Comment:
Does this very useful tip only apply to precision landing? Could it be moved up to the main page?

Reply:
no,it only applies to preflare which requires airspeed sensor